core/ordering/cosipbft/blocksync/default_test.go

```diff
@@ -182,15 +189,21 @@ func TestHandler_Stream(t *testing.T) {
 	handler.pbftsm = testSM{blocks: handler.blocks}
```

Comment:
Would it make sense to split this in small tests ? :)

Reply:
I'm doing this progressively when updating the documentation.